ui/about.go

```diff
@@ -30,6 +51,8 @@ func (h *handler) showAboutPage(w http.ResponseWriter, r *http.Request) {
 	view.Set("user", user)
 	view.Set("countUnread", h.store.CountUnreadEntries(user.ID))
 	view.Set("countErrorFeeds", h.store.CountUserFeedsWithErrors(user.ID))
+	view.Set("showOptions", showOptions)
+	view.Set("options", options)
```

Comment:
`options` should be more explicit. I would replace with `configOptions` or `globalConfigOptions`.